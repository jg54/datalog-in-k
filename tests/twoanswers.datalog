test("a", "b") :- .
test(1, "t") :- .
test(Y, X)?