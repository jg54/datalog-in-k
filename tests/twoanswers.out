test("a","b")
test(1,"t")
